CHANGELOG.txt

```diff
@@ -1,5 +1,98 @@
+Airflow 2.0.2, 2021-04-19
+-------------------------
+
+Bug Fixes
+"""""""""
+
+* Fix missing on_load trigger for folder-based plugins (#15208)
+* ``kubernetes cleanup-pods`` subcommand will only clean up Airflow-created Pods (#15204)
+* Fix password masking in CLI action_logging (#15143)
+* Fix url generation for TriggerDagRunOperatorLink (#14990)
+* Restore base lineage backend (#14146)
+* Unable to trigger backfill or manual jobs with Kubernetes executor. (#14160)
+* Bugfix: Task docs are not shown in the Task Instance Detail View (#15191)
+* Bugfix: Fix overriding `pod_template_file` in KubernetesExecutor (#15197)
```

Comment:
```suggestion
* Bugfix: Fix overriding ``pod_template_file`` in ``KubernetesExecutor`` (#15197)
```

Reply:
Does KubeExecutor need to be code-ified?

Reply:
not strictly needed tbh -- just looks nice :D http://airflow.apache.org/docs/apache-airflow/stable/changelog.html